recipes/Semi-ATE/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "Semi-ATE" %}
+{% set version = "0.1.27" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/Semi-ATE-{{ version }}.tar.gz
+  sha256: 8efe206226cc62752e04b084b2e4b95cbbf445f0f0363208842bfadd3765c45e
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - python >=3.7
+
+test:
+  imports:
+    - ATE
```

Comment:
These imports don't actually trigger anything that verifies the package is working. Can you add some imports that will fail if all the right dependencies aren't installed, etc.?

Reply:
@dopplershift  a bit of background : 

Semi-ATE is a plugin for Spyder (>= 5.1.0) and it is sponsored by TDK. It enables an IDE (Spyder thus) to handle so called ATE test programs. (read: software that uses instruments to test semiconductor chips ... hence the name `Semi-ATE`)

As Spyder is Qt based, also our plugin is Qt based and implementing meaningful tests is not trivial to begin with in the first place, but more importantly Qt on conda-forge is currently old, semi-broken and doesn't support ARM (for neither Linux or macOS) ...

That's why we sponsored the Spyder team (we work closely together with them) to upgrade the Qt package on conda-forge at least to the latest 5-series and for both Intel and ARM. This work is done, and is in the process of being reviewed/documented with the relevant conda-forge responsibles. It is expected that early October this will become available on conda-forge.

Once we have a state-of-the-art Qt in place, we plan (Q1/22) to split the test suit for Semi-ATE we have on our internal servers into: 
- Graphical tests ➜ will be implemented on OUR public GitHub with GitHub actions instead as now on internal servers with Jenkins.
- Functional tests ➜ will be implemented IN the module

This way we can do something like: 
 
```
import ATE
ATE.test
```
which would then run the whole __functional test suit__ (read: non-graphical test suit) inspired by packages as numpy, bitarray and the likes. Which is in the end what you are asking for :smile: 

But wait, there is more! 

We also have a project called [maxiconda](https://github.com/Semi-ATE/maxiconda) which is basically a conda installer purely based on conda-forge. 

We use `maxiconda` to show our patrons what we do with their $$$. (as they don't like the new `terms of use` of Anaconda AT ALL ... so I promised them `maxiconda` which is-and-always-will-be free for everyone regardless if they are a hobbyist, small company, medium company, large company or multinational!) Anyway, point being, that we promote conda-forge toward TDK, but what they are really looking for is Semi-ATE so I need to get Semi-ATE on conda-forge ASAP ...

I would thus appreciate if you could wave this first iteration of Semi-ATE through without the test at this moment, so we can please our patron and get the next budget to further contribute to Spyder, Qt and conda-forge.

I promise that with the next budget we will implement a (very) comprehensive test suit as outlined above! (it is in our own interest anyway)

Thanx in advance,

Tom

Reply:
Fair enough.

Reply:
Thanx!